pkg/cmd/step/step_tag_test.go

```diff
@@ -88,6 +87,34 @@ func TestStepTagCharts(t *testing.T) {
 		}
 	}
 
-	assert.True(t, foundRepo, "Failed to find tag '%s' in file %s", step.ValuesYamlRepositoryPrefix, valuesFile)
-	assert.True(t, foundVersion, "Failed to find tag '%s' in file %s", step.ValuesYamlTagPrefix, valuesFile)
+	assert.True(t, foundRepo, "Failed to find tag '%s' in file %s", ValuesYamlRepositoryPrefix, valuesFile)
+	assert.True(t, foundVersion, "Failed to find tag '%s' in file %s", ValuesYamlTagPrefix, valuesFile)
+}
+
+func TestDefaultChartValueRepositoryValidName(t *testing.T) {
+	f, err := ioutil.TempDir("", "test-step-tag-valid-name")
```

Comment:
I would clean up the temp dir when the test is done with:

```
defer os.RemoveAll(f)
```

Reply:
Quite agreed but that wasn't cleaned neither in the other tests of the file, I assumed there was a good reason for this. 

I'll fix that next week when I'm back :)